app/controllers/landing_page_controller.rb

```diff
@@ -136,6 +136,196 @@ def data_str
       "signup_button_color_hover": {"type": "marketplace_data", "id": "primary_color_darken"}
     },
     {
+      "id": "categories7",
+      "kind": "categories",
+      "title": "Section title goes here",
+      "paragraph": "Section paragraph goes here",
+      "button_color": {"type": "marketplace_data", "id": "primary_color"},
+      "button_color_hover": {"type": "marketplace_data", "id": "primary_color_darken"},
+      "button_title": "Section link",
+      "button_path": {"value": "https://google.com"},
+      "category_color_hover": {"type": "marketplace_data", "id": "primary_color"},
+      "categories": [
+        {
+          "title": {"value": "Mountain bikes"},
+          "background_image": {"type": "assets", "id": "myheroimage"},
+          "path": {"value": "https://google.com"}
+        },
```

Comment:
The linking could be done so that editor user puts in something like:
```json
"categories": [ {"type": "category", "id": 1234}, ... ]
```

which then gets expanded to full category objects like this:

```json
"categories: [
  { "type": "category",
    "id": 1234,
    "title": "foobardoodoodo",
    "background_image": "...",
    "path": "..."
  }, ...
]
```